server/src/main/scala/net/wiringbits/apis/EmailApiAWSImpl.scala

```diff
@@ -23,8 +26,12 @@ class EmailApiAWSImpl @Inject() (
       s"""<p>${emailRequest.message.body}</p>""".stripMargin
 
     def unsafe(): Unit = try {
+      val region = Regions.fromName(awsConfig.region)
```

Comment:
Let's update `AWSConfig` to invoke `Regions.fromName` there, there is no need to parse every time we need it